lib/workers/repository/update/branch/index.spec.ts

```diff
@@ -1705,5 +1705,65 @@ describe('workers/repository/update/branch/index', () => {
         'No package files need updating'
       );
     });
+
+    it('Dependency Dashboard All Pending approval', async () => {
+      getUpdated.getUpdatedPackageFiles.mockResolvedValueOnce({
```

Comment:
instead of 
jest.mock('./get-updated');
const getUpdated = mocked(_getUpdated);

try jest.spyOn( getUpdated , 'getUpdatedPackageFiles').mockResolvedValueOnce